test/controllers/NotificationControllerSpec.scala

```diff
@@ -273,6 +274,60 @@ class NotificationControllerSpec extends PlaySpec with MockitoSugar with ScalaFu
 
     }
 
+    "return correct Content Type" when {
```

Comment:
here as well -return correct  contact type and template version

Reply:
yeah I'll make the change.